Modules/Devices/test/Devices.Infrastructure.Tests/TestDataGenerator.cs

```diff
@@ -14,17 +14,20 @@ public static Device CreateDevice(string? language = null)
         return CreateIdentityWithOneDevice(language).Devices.First();
     }
 
-    public static Identity CreateIdentityWithOneDevice(string? deviceCommunicationLanguage = null)
+    public static Identity CreateIdentityWithOneDevice(string? language = null)
     {
+        var deviceCommunicationLanguage = language != null ? CommunicationLanguage.Create(language).Value : CommunicationLanguage.DEFAULT_LANGUAGE;
+
         var identity = new Identity(
             CreateRandomDeviceId(),
             CreateRandomIdentityAddress(),
             CreateRandomBytes(),
             CreateRandomTierId(),
-            1
+            1,
+            deviceCommunicationLanguage
         );
 
-        identity.Devices.Add(new Device(identity, deviceCommunicationLanguage != null ? CommunicationLanguage.Create(deviceCommunicationLanguage).Value : CommunicationLanguage.DEFAULT_LANGUAGE));
+        identity.Devices.Add(new Device(identity, deviceCommunicationLanguage));
```

Comment:
This shouldn't actually be necessary, because a new identity automatically has 1 device now.

Reply:
done.